doc/en/announce/release-5.3.2.rst

```diff
@@ -0,0 +1,27 @@
+pytest-5.3.2
+=======================================
+
+pytest 5.3.2 has just been released to PyPI.
+
+This is a bug-fix release, being a drop-in replacement. To upgrade::
+
+  pip install --upgrade pytest
+
+The full changelog is available at https://docs.pytest.org/en/latest/changelog.html.
+
+Thanks to all who contributed to this release, among them:
+
+* Anthony Sottile
+* Bruno Oliveira
+* Claudio Madotto
+* Daniel Hahler
+* Jared Vasquez
+* Michael Rose
+* Ran Benita
+* Ronny Pfannschmidt
+* Zac Hatfield-Dodds
+* zupermanzupereroe
```

Comment:
You can remove zupermanzupereroe here, I already added my real name few lines above.

Reply:
Done, thanks!